File: c/primes.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
	int k;
  int cnt=0;
	for (k = 2; k < 100000; k++)
	{
		if (isPrime(k))
            cnt++;
	}

  printf("found %d prime numbers",cnt);
	exit(0);
}

int isPrime(int number)
{
   int i;
   for (i = 2; i < number; i++ )
   {
       if (number % i == 0)
       {
           return 0;
       }
   }
   return 1;
}

File: lua/primes.lua
function isPrime(number)
   for i = 2, number - 1, 1 do
        if number % i == 0 then
           return 0
        end
   end
   return 1
end

local cnt = 0
for k = 2, 100000,1 do
	if isPrime(k) == 1 then
          cnt = cnt + 1
  end
end
print(cnt)
